scripts/infra/nvidia-setup.sh

```diff
@@ -113,15 +113,15 @@ fi \
     && cp -a /etc/dnf/dnf.conf{,.tmp} && mv /etc/dnf/dnf.conf{.tmp,} \
     && dnf config-manager --best --nodocs --setopt=install_weak_deps=False --save \
     && dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel${OS_VERSION_MAJOR}/${CUDA_REPO_ARCH}/cuda-rhel${OS_VERSION_MAJOR}.repo \
-    && dnf -y module enable nvidia-driver:${DRIVER_STREAM}/default \
+    && dnf -y module enable nvidia-driver:${DRIVER_STREAM}-dkms/default \
     && export NCCL_PACKAGE=$(dnf search libnccl --showduplicates 2>/dev/null | grep ${CUDA_MAJOR_MINOR} | awk '{print $1}' | grep libnccl-2 | tail -1) \
     && dnf install -y \
         cloud-init \
         pciutils \
         tmux \
         nvidia-driver-cuda-${DRIVER_VERSION} \
         nvidia-driver-libs-${DRIVER_VERSION} \
-        nvidia-driver-NVML-${DRIVER_VERSION} \
+        libnvidia-ml-${DRIVER_VERSION} \
```

Comment:
I understand the DKMS changes since you need the DKMS versions to support multiple kernels, but I am not quite understanding why we removing the NVML driver package and replacing it with `libnvidia-ml`. Do we not need  `nvidia-driver-NVML` anymore? I know that `libnvidia-ml` will install NVML, which is necessary for certain `nvidia-smi` functionality. Were we simply pulling in `nvidia-driver-NVML` this whole time when we should have been pulling in `livnvidia-ml` specifically? Just making sure we don’t “need” this particular driver package.

Reply:
Also perhaps helpful if you haven't seen it in the commit message: https://github.com/NVIDIA/yum-packaging-nvidia-driver/blob/d06f50a507eb6e053e2e7dd4fcd7781060f8ff06/nvidia-driver.spec#L117